app/src/main/java/org/mozilla/focus/urlinput/UrlInputFragment.java

```diff
@@ -241,7 +241,7 @@ public void setSuggestions(@Nullable List<CharSequence> texts) {
             return;
         }
 
-        final String searchKey = urlView.getOriginalText().trim();
+        final String searchKey = urlView.getOriginalText().trim().toLowerCase();
```

Comment:
Looks like a good solution to me. I've got some other things in my mind, though:

A foreseeable situation I can see is this will not work when the user is using Turkish locale. which has a special lowercase of I : ı https://en.wikipedia.org/wiki/Dotted_and_dotless_I

so I would recommend:
final String searchKey = urlView.getOriginalText().trim().toLowerCase(Locale.US);

Another situation is, I believe some search suggestions are provided in capital: I haven't found an example in Google's suggestions but you can try "CD" for duckduckgo, this solution also break under suggestions like this. I'm not sure if lowering every suggestion sounds like a good idea. I'll leave it to the reviewer to decide :P @walkingice can you help review this?